R/package_update.R

```diff
@@ -29,26 +29,26 @@ update_package <- function(package = NULL, name = NULL, path = getwd()) {
   }
 
```

Comment:
Need to be clear in the function description which files will get updated, if necessary, which files may need to be updated manually, and maybe even which files contributors can rely on not being touched?